sentry/src/test/java/io/sentry/SentryContext.kt

```diff
@@ -0,0 +1,23 @@
+package io.sentry
+
+import kotlinx.coroutines.ThreadContextElement
+import kotlin.coroutines.AbstractCoroutineContextElement
+import kotlin.coroutines.CoroutineContext
+
+class SentryContext : ThreadContextElement<IHub>, AbstractCoroutineContextElement(Key) {
```

Comment:
This is done in a similar way to how slf4j support is implemented in https://github.com/Kotlin/kotlinx.coroutines/blob/master/integration/kotlinx-coroutines-slf4j/src/MDCContext.kt